src/or/dirvote_common.c

```diff
@@ -182,8 +182,8 @@ dirvote_recalculate_timing(const or_options_t *options, time_t now)
 /** Return the signature made by <b>voter</b> using the algorithm
  * <b>alg</b>, or NULL if none is found. */
 document_signature_t *
-voter_get_sig_by_algorithm(const networkstatus_voter_info_t *voter,
-                           digest_algorithm_t alg)
+dirvote_get_voter_sig_by_alg(const networkstatus_voter_info_t *voter,
```

Comment:
Maybe this function should be networkstatus_voter_get_sig_by_alg?  It works  on a networkstatus_voter_info_t.

Reply:
The `dirvote_` is simply to namespace it to the file which `networkstatus_` would fail that with the current filename.

However, I do see `networkstatus_get_voter_by_id()` so maybe we should simply move that function into `networkstatus.c` ?

Reply:
Yes, let's.  We can do it postmerge.

Reply:
Ack.